docs/02-formalia/titlepage.rst

```diff
@@ -5,31 +5,164 @@
    :alt: eProsima
    :target: http://www.eprosima.com/
 
-*eprosima Fast RTPS* is a C++ implementation of the RTPS (Real Time Publish-Subscribe) protocol, which provides
-publisher-subscriber communications over unreliable transports such as UDP,
-as defined and maintained by the Object Management Group (OMG) consortium.
-RTPS is also the wire interoperability protocol defined for the Data Distribution Service (DDS) standard, again by the
-OMG.
-*eProsima Fast RTPS* holds the benefit of being standalone and up-to-date, as most vendor solutions either implement
-RTPS as a tool to implement DDS or use past versions of the specification.
-
-Some of the main features of this library are:
-
-* Configurable best-effort and reliable publish-subscribe communication policies for real-time applications.
-* Plug and play connectivity so that any new applications are automatically discovered by any other members of the
-  network.
-* Modularity and scalability to allow continuous growth with complex and simple devices in the network.
-* Configurable network behavior and interchangeable transport layer: Choose the best protocol and system input/output
-  channel combination for each deployment.
-* Two API Layers: a high-level Publisher-Subscriber one focused on usability and a lower-level Writer-Reader one that
-  provides finer access to the inner workings of the RTPS protocol.
-
-eProsima Fast RTPS has been adopted by multiple organizations in many sectors including these important cases:
-
-* Robotics: ROS (Robotic Operating System) as their default middleware for ROS2.
-* EU R&D: FIWARE Incubated GE.
-
-This documentation is organized into the following sections:
+*eprosima Fast DDS* is a C++ implementation of the
+`DDS (Data Distribution Service) Specification <https://www.omg.org/spec/DDS/About-DDS/>`__, a protocol
+defined by the Object Management Group (OMG).
+The *eprosima Fast DDS* library provides both an Application Programming Interface (API) and a communication protocol
+that deploy
+a Data-Centric Publisher-Subscriber (DCPS) model, with the purpose of establishing efficient and reliable
+information distribution among Real-Time Systems.
+*eprosima Fast DDS* is predictable, scalable, flexible, and efficient in resource handling.
+For meeting these requirements, it makes use of typed interfaces and hinges on a many-to-many
+distributed network paradigm that neatly allows separation of the publisher and subscriber sides of the communication.
+
+*eprosima Fast DDS* comprises:
+
+* The DDS API implementation.
+* A generation tool for bridging typed interfaces with the middleware implementation.
+* The underlying wire protocol implementation.
+
+.. figure:: /01-figures/DDS_concept_vertical.svg
```

Comment:
I think the figure is out of context or explanation here. I'd move to right after the DDS API text, and maybe add a caption in the `figure` directive, as explained [here](http://openalea.gforge.inria.fr/doc/openalea/doc/_build/html/source/sphinx/rest_syntax.html?highlight=figure#figure-directive)